modular_citadel/code/modules/client/loadout/backpack.dm

```diff
@@ -115,4 +115,17 @@
 	name = "A fancy pen"
 	category = SLOT_IN_BACKPACK
 	path = /obj/item/pen/fountain
-	cost = 2
\ No newline at end of file
+	cost = 2
+
+/datum/gear/modular_tablet
+	name = "A modular tablet"
+	category = SLOT_IN_BACKPACK
+	path = /obj/item/modular_computer/tablet/preset/cheap/
+	cost = 4
+
+/datum/gear/modular_laptop
+	name = "A modular laptop"
+	category = SLOT_IN_BACKPACK
+	path = /obj/item/modular_computer/laptop/preset/civillian
```

Comment:
i was gonna complain about "civillian" but it's actually called that in the code. neat. change it while you're at it actually maybe please.

Reply:
Alright